### PR TITLE
Add FencerDevLog to tutorials

### PR DESCRIPTION
**Pull request by FilipRachunek**

### CODE REVIEW

File: community/tutorials.rst [hunk 1]
```diff
@@ -46,6 +46,7 @@ Video tutorials
 - `Gwizz <https://www.youtube.com/@Gwizz1027>`_ (2D, GDScript).
 - `Quiver <https://quiver.dev/>`_ (2D, GDScript).
 - `Maker Tech <https://www.youtube.com/watch?v=0mUoRdYe0s4>`_ (2D, GDScript).
+- `FencerDevLog <https://www.youtube.com/@FencerDevLog>` _ (2D, 3D, GDScript).
```
Comment (AThousandShips) [inline community/tutorials.rst:49]:
```suggestion
- `FencerDevLog <https://www.youtube.com/@FencerDevLog>`_ (2D, 3D, GDScript).
```

File: community/tutorials.rst [hunk 2]
```diff
@@ -46,6 +46,7 @@ Video tutorials
 - `Gwizz <https://www.youtube.com/@Gwizz1027>`_ (2D, GDScript).
 - `Quiver <https://quiver.dev/>`_ (2D, GDScript).
 - `Maker Tech <https://www.youtube.com/watch?v=0mUoRdYe0s4>`_ (2D, GDScript).
+- `FencerDevLog <https://www.youtube.com/@FencerDevLog>` _ (2D, 3D, GDScript, Shaders).
```
Comment (AThousandShips) [inline community/tutorials.rst:49]:
```suggestion
- `FencerDevLog <https://www.youtube.com/@FencerDevLog>`_ (2D, 3D, GDScript, Shaders).
```

Comment (AThousandShips) [reply community/tutorials.rst:49]: You keep ignoring this change and marking it as resolved, why? Are you having problems pushing your changes?

Comment (FilipRachunek) [reply community/tutorials.rst:49]: Sorry, I am not certain what to do here. Is there a problem with the space? I must have overlooked it.

Comment (AThousandShips) [reply community/tutorials.rst]: You were instructed twice for this? And you marked it as resolved, so you must have seen it right?

**Comment by AThousandShips** [comment]
Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

File: community/tutorials.rst (hunk 2 above)
Comment (AThousandShips) [reply community/tutorials.rst]: If you don't know what to do with a suggestion please ask instead of marking it as resolved 🙂

**Comment by FilipRachunek** [comment]
I see. Sorry, I don't know how to squash commits, my knowledge of Git is very basic, and I hardly use the command line.
Unless there is a better option, I will probably close this pull request, and create a new one.

**Pull request closed without merging**